ci-operator/config/opendatahub-io/codeflare-operator/opendatahub-io-codeflare-operator-main.yaml

```diff
@@ -48,19 +51,20 @@ tests:
         git clone https://github.com/opendatahub-io/distributed-workloads.git
         cd distributed-workloads
         make install-opendatahub-operator
-        # To be removed once https://github.com/opendatahub-io/opendatahub-operator/issues/600 is implemented and new ODH version is released
-        make install-codeflare-operator
         if [[ "${REPO_OWNER}" == "openshift" ]]; then
           echo -e "Running in openshift repo, it means that this is PJ rehearsal, using latest main for CodeFlare manifest \n"
-          sed -i "s|<DW PR tarball URI>|https://api.github.com/repos/opendatahub-io/distributed-workloads/tarball/main|" ./contrib/configuration/basic-dsc.yaml
+          sed -i "s|<CFO PR tarball URI>|https://api.github.com/repos/opendatahub-io/codeflare-operator/tarball/main|" ./contrib/configuration/basic-dsc.yaml
         else
           echo -e "Running PR ${PULL_NUMBER} \n"
-          sed -i "s|<DW PR tarball URI>|https://api.github.com/repos/opendatahub-io/distributed-workloads/tarball/pull/${PULL_NUMBER}/head|" ./contrib/configuration/basic-dsc.yaml
+          sed -i "s|<CFO PR tarball URI>|https://api.github.com/repos/opendatahub-io/codeflare-operator/tarball/pull/${PULL_NUMBER}/head|" ./contrib/configuration/basic-dsc.yaml
         fi
         oc apply -f ./contrib/configuration/basic-dsc.yaml -n opendatahub
         cat ./contrib/configuration/basic-dsc.yaml
         sleep 60
         cd tests && go test -timeout 60m -v ./integration
+      dependencies:
+      - env: ODH-CODEFLARE-OPERATOR-CONTROLLER-IMAGE
+        name: image-registry.openshift-image-registry.svc:5000/openshift-operators/codeflare-operator
```

Comment:
```suggestion
        name: codeflare-operator
```

Reply:
Yeah I just realized that, good catch thanks!